### ISSUE TITLE
Promote Account Console v3 to 'preview'

### ISSUE THREAD

**Issue by jonkoops** [keycloak/keycloak#19659]
The new Account Console introduced under #19641 must be promoted to 'preview' so that more users will have access to it by default.

**Comment by edewit** [comment]
done

**Comment by jonkoops** [comment]
@edewit No, it's not. It's still [marked as experimental](https://github.com/keycloak/keycloak/blob/cd9dce68dad25df1a9a30edb9b03a76d8b78161f/common/src/main/java/org/keycloak/common/Profile.java#L48).
